api/build.gradle

```diff
@@ -1,7 +1,7 @@
 buildscript {
     ext {
         junitVersion = '5.3.2'
-        jacksonDataformatVersion = '2.9.9'
+        jacksonDataformatVersion = '2.9.10'
```

Comment:
2.10.0 is available

Reply:
Yes, but that version doesn't work with the ApiDocumentation without further changes which I wasn't prepared to do in this PR.